docs/sources/enterprise/usage-insights.md

```diff
@@ -0,0 +1,53 @@
++++
+title = "Usage-insights"
+description = "Usage-insights"
+keywords = ["grafana", "usage-insights", "enterprise"]
+aliases = ["/docs/grafana/latest/enterprise/usage-insights/"]
+type = "docs"
+[menu.docs]
+name = "Usage-insights"
+parent = "enterprise"
+weight = 700
++++
+
+# Usage insights
+
+Usage insights allows you to have a better understanding of how your Grafana instance is used. The collected data are: dashboard views (aggregated and per user), data source errors, and number of data source queries.
+
+> Only available in Grafana Enterprise v7.0+.
+
+## Presence indicator
+
+Presence indicator is visible to all signed-in users on all dashboards. It shows the avatars of users who interacted with the dashboard recently. You can see the user's name by hovering your cursor over the user's avatar. The avatars come from [Gravatar](https://gravatar.com) based on the user's email.
+
+When more users are active on a dashboard than can fit in the presence indicator section, click on the `+X` icon that opens [dashboard insights]({{< relref "#dashboard-insights" >}}) to see more details about recent user activity. 
+
+{{< docs-imagebox img="/img/docs/enterprise/presence_indicators.png" max-width="400px" class="docs-image--no-shadow" >}}
+
+You can choose your own definition of "recent" by setting it in the [configuration]({{< relref "../installation/configuration.md">}}) file.
+
+```ini
+[analytics.views]
+# Set age for recent active users
+recent_users_age = 24h
```

Comment:
```suggestion
recent_users_age = 15m
```

It's a good idea to use the default value in examples unless changing the value is required to understand how to change it (as in white labelling footer links).